ui/src/app/app-routing.module.ts

```diff
@@ -286,30 +287,44 @@ const routes: Routes = [
           }]
       },
       {
-        path: 'admin/review-workspace',
-        component: AdminReviewWorkspaceComponent,
-        data: {title: 'Review Workspaces'}
-      }, {
-        path: 'admin/user',
-        component: AdminUserComponent,
-        data: {title: 'User Admin Table'}
-      }, {
-        path: 'admin/banner',
-        component: AdminBannerComponent,
-        data: {title: 'Create Banner'}
-      }, {
-        path: 'admin/workspaces',
-        component: AdminWorkspaceSearchComponent,
-        data: { title: 'Workspace Admin'},
-      }, {
-        path: 'admin/workspaces/:workspaceNamespace',
-        component: AdminWorkspaceComponent,
-        data: { title: 'Workspace Admin'}
-      }, {
+        path: 'admin',
+        children: [{
+          path: 'review-workspace',
+          component: AdminReviewWorkspaceComponent,
+          data: {title: 'Review Workspaces'}
+        },
+          {
+            path: 'user',
+            component: AdminUsersComponent,
+            data: {title: 'User Admin Table'}
+          },
+          {
+            path: 'user/:userId',
```

Comment:
Separate comment on this line: I think the URL path param should be username, not user ID. I'd advocated for this in the original [design doc](https://docs.google.com/document/d/1kuo-Q6qoK8Y9_bMKppwm11_n7FcU9d_JGwVEv9ZaouM/edit#heading=h.wzn542n9elt) – it's important because we may want external systems to be able to create inbound links to our admin pages, so we should use externally-meaningful URL paths wherever possible.

Reply:
The username is a `research-aou.org` email address. What about everything in front of the @?